moveit_ros/planning/moveit_cpp/include/moveit/moveit_cpp/planning_component.h

```diff
@@ -154,19 +188,30 @@ class PlanningComponent
   /** \brief Set the path constraints generated from a moveit msg Constraints */
   bool setPathConstraints(const moveit_msgs::msg::Constraints& path_constraints);
 
+  /** \brief Set the trajectory constraints generated from a moveit msg Constraints */
+  bool setTrajectoryConstraints(const moveit_msgs::msg::TrajectoryConstraints& trajectory_constraints);
```

Comment:
What the heck is `trajectory_constraints` vs `path_constraints`? The only difference is timing and vel/accel information, but the motion planners aren't concerned with that. So I don't get what this function does that `setPathConstraints()` can't do.

Reply:
I simply tried to make the interface consistent since the MotionPlanRequest differs between path and trajectory constraints. 

I'd be happy to simplify the interface but I think we should do this in a follow up PR.